8.x/theme-development/file-management.md

```diff
@@ -0,0 +1,147 @@
+
+In Laravel, `@yield` is a blade directive that defines a section of content that can be replaced by child views. The `@yield` directive is typically used in a layout file to define areas of the page that can be customized by other views that extend the layout.
+
+For example, let's say you have a layout file called `_base.blade.php` that includes a `@yield('title')` directive. You can then create a child view called `index.blade.php` that extends the layout and includes its own content within the content section:
+
+```html
+// _base.blade.php
+<html>
+    <head>
+        <title>@yield('title') My website</title>
+    </head>
+    <body>
+        ...
+    </body>
+</html>
+
+// index.blade.php
+
+@extends('_base')
+
+@section('title')
+    index page
+@endsection
+```
+When you render the `index.blade.php` view, Laravel will replace the `@yield('content')` directive in the `_base.blade.php` layout with the content defined in the `@section('title')` directive in the child view, resulting in the following HTML output:
+
+```html
+<html>
+    <head>
+        <title>index page My website</title>
+    </head>
+    <body>
+        ...
+    </body>
+</html>
+```
+
+or another example can be :
+
+```html
+// _base.blade.php
+<html>
+    <head>
+    ...
+    </head>
+    <body>
+      <header class="@yield('header_class')">
+        ....
+    </body>
+</html>
+
+// index.blade.php
+
+@extends('_base')
+
+@section('header_class')
+    Home
+@endsection
+
+```
+In this example, the parent view `(_base.blade.php)` has a header element with a class attribute whose value is determined by the `@yield("header_class")` directive.
+This means that when the child view `(index.blade.php)` is rendered, the `Home` string will be placed in the `@yield("header_class")` section of the parent view, resulting in the header element having a class attribute of "Home".
+
+
+### How to use components
+
+When you use the `@include` directive, Blade will render the specified view and insert its contents into the parent view at the point where the `@include` directive is used. This allows you to reuse code across multiple views by breaking it up into smaller, reusable components.
+
+Assume you have a file named `_underscore-template.blade.php` and you want to use its component within your page. Using this directive allows you to do so without making it overcrowded.
+
+Example:
+
+```html
+@include('_underscore-template')
+```
+
+**Note:** Consider that it's reasonable to put all of the repeatable components in the `_base.blade.php` to save the time and ebnergy for designing new pages.
+
+
+### How to review changes
+
+To review the changes on the files, `/path/to/larammerce-base-theme`, run `npm run watch` command, and after every change, run `./deploy` to see the results.
+
+If you are adding a new file:
+
+ - Firstly, `/path/to/resourses/assets/sass/` and create a file. (for example  `page-index.scss`).

```

Comment:
/path/to/resourses/assets/sass/  ->  /path/to/larammerce-base-theme/resourses/assets/sass/